File: cw/java/ReturningStrings.java
// https://www.codewars.com/kata/55a70521798b14d4750000a4/train/java


import java.util.Scanner;

public class ReturningStrings {
    public static String greet(String name) {
        return "Hello, " + name + " how are you doing today?";
    }

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String name = scan.nextLine();
        scan.close();

        System.out.println(greet(name));
    }
}

File: cw/cpp/returning_strings.cpp
// https://www.codewars.com/kata/55a70521798b14d4750000a4/train/cpp


#include <iostream>

std::string greet(const std::string& n) {
    return "Hello, " + n + " how are you doing today?";
}

int main() {
    std::string n;

    std::cin >> n;
    std::cout << greet(n) << std::endl;

    return 0;
}